Autopilot/Inc/SDCard.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * SDcard fuctions
+ * Author: Avani Joshi and Laye Mehta
+ */
+
+#ifndef SDCARDHEADER_HPP
+#define SDCARDHEADER_HPP
+
+#include "SPI.hpp"
+
+struct SDCardData_t {
+	
+	bool isRead; 
+ 
+    bool status; //Success = 1 and Fail = 0
+};
+
+
+class SDCard{
+    public:
+	
+	/*
+	THE FUNCTION ARGUMENTS WILL BE ADDED AS THE IMPLEMENTATION IS CREATED
+	*/
+		
+		/*
+		Reads from SD Card
+		*/
+		virtual void Read() = 0; 
+		
+		/*
+		Writes to SD Card
+		*/
+		virtual void Program() = 0; 
```

Comment:
Curious, do you intend to make another class that will inherit these functions? I don't think an individual card requires its own implementation but I could be wrong since I haven't researched SD cards in-depth 😅